modules/renter/hostdb/weightedlist_test.go

```diff
@@ -326,12 +329,6 @@ func TestRandomHosts(t *testing.T) {
 	if len(randHosts) != 3 {
 		t.Error("didn't get 3 hosts")
 	}
-	if len(hdb.activeHosts) != 3 {
-		t.Error("wrong number of hosts")
-	}
-	if hdb.hostTree.weight.Cmp(types.NewCurrency64(6)) != 0 {
-		t.Error("unexpected weight at initialization")
-	}
```

Comment:
they're not really testing the `RandomHosts` function, just the implementation details of the host tree.